modules/social.py

```diff
@@ -133,6 +133,31 @@ def __init__(self, bot):
             'Most likely',
         ]
 
+        self.triviaTrophyIndex = {
```

Comment:
These are *okay* but really if your going to have your keys be `0, 1, 2, ... 9` -- an array would have been a better choice.

For example, later in the code, you could have replaced this line of code:
`currentLevel = [key for key, value in self.triviaTrophyIndex.items() if value == t][0]`
with this if you used an list:
`currentLevel = self.triviaTrophyIndex.index(t)`